src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -366,7 +369,7 @@ public void updateKernelConfigWithIotConfiguration(Kernel kernel, ThingInfo thin
         }
 
         Path caFilePath = certPath.resolve("rootCA.pem");
-        downloadRootCAToFile(caFilePath.toFile());
+        downloadCAToFile(caFilePath.toFile(), ROOT_CA_URL, CA_3_URL);
```

Comment:
are we downloading both CA1 and CA3 certs all the time? is that safe or a best practice?